packages/monitor-v2/src/monitor-polymarket/index.ts

```diff
@@ -1,50 +1,19 @@
 import { delay } from "@uma/financial-templates-lib";
-import { BotModes, initMonitoringParams, Logger, startupLogLevel, waitNextBlockRange } from "./common";
+import { initMonitoringParams, Logger } from "./common";
 import { monitorTransactionsProposedOrderBook } from "./MonitorProposalsOrderBook";
 
 const logger = Logger;
 
 async function main() {
   const params = await initMonitoringParams(process.env);
-  const cache = { ancillaryData: new Map<string, string>() };
 
-  logger[startupLogLevel(params)]({
+  logger.debug({
     at: "PolymarketMonitor",
     message: "Polymarket Monitor started 🔭",
-    botModes: params.botModes,
   });
 
-  const cmds = {
-    transactionsProposedEnabled: monitorTransactionsProposedOrderBook,
-  };
-
```

Comment:
why was this removed? I think we want the ability to run the bot in _both_ looping mode _and_ serverless mode, no? for example we want to enable other community members or others to run the bot within a normal vm. this is how all other UMA/Across infra is structured.

Reply:
I see. I removed it because this bot is not meant to be run in our infra like that (that's what initially confused me when I thought I could run them in loops)... But you are right. Let me undo this changes so we can support this way of running it.